autoload/ale/handlers/eslint.vim

```diff
@@ -82,6 +82,10 @@ function! ale#handlers#eslint#Handle(buffer, lines) abort
         let l:type = 'Error'
         let l:text = l:match[3]
 
+        if l:text == 'File ignored because of a matching ignore pattern. Use "--no-ignore" to override.'
```

Comment:
You should install Vint for checking your code. It will complain about the use of `==`. I am also banning `==#` and `==?` from the codebase, as they are also a frequent source of bugs. I recommend using `x is# y` instead.